frozen-abi/Cargo.toml

```diff
@@ -42,8 +42,8 @@ im = { workspace = true, features = ["rayon", "serde"] }
 memmap2 = { workspace = true }
 # These dependencies are used only to back `frozen-abi` StableAbi API,
 # to avoid version skew and keep digest computation stable regardless of consumer dependencies.
-rand = "0.8.5"
-rand_chacha = "0.3.1"
+rand = { workspace = true }
```

Comment:
Should we pin the version of rand here to avoid ABI hashes from breaking accidentally? None of this runs in prod, so perf is a non-issue.

Reply:
How different would they be from these? https://github.com/anza-xyz/solana-sdk/blob/master/scripts/test-frozen-abi.sh